Dockerfile

```
@@ -1,26 +1,51 @@
+ARG VERSION=unspecified
+
+FROM python:3.6-slim-buster
+
+ARG VERSION
+
+###
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
+# Note: Additional labels are added by the build workflow.
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
+LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
+
 ###
-# Install everything we need
+# Unprivileged user setup variables
 ###
-FROM python:3.6-slim-buster AS install
-LABEL maintainer="jeremy.frasier@trio.dhs.gov"
-LABEL organization="CISA Cyber Assessments"
-LABEL url="https://github.com/cisagov/scanner"
+ARG CISA_UID=421
```

This aligns with the changes in https://github.com/cisagov/cyhy_amis/pull/559 and I've got similar changes in place for two other Docker images designed to run in the CyHy environment.

```suggestion
ARG CISA_UID=2048
```

---

We should probably change it in the skeleton then, no?

---

This is only necessary for images running in the CyHy environment that need access to restricted files.

---

Change UID to 2048.